cms/Cargo.toml

```diff
@@ -22,9 +22,12 @@ x509-cert = { version = "=0.3.0-pre.0", default-features = false }
 
 # optional dependencies
 aes = { version = "=0.9.0-pre.2", optional = true }
+aes-kw = { version ="0.2.1", optional = true }
```

Comment:
This duplicates the dependency on aes and cipher, we should bump aes-kw to use "aes 0.9.0-pre.2" & "cipher 0.5.0-pre.7".

Reply:
https://github.com/RustCrypto/key-wraps/pull/34

Reply:
https://github.com/nemynm/formats/pull/1

Reply:
Thanks, I'll merge once it's ready

Reply:
Merged